packages/discord.js/typings/index.d.ts

```diff
@@ -968,7 +968,7 @@ export abstract class BaseChannel extends Base {
   public get url(): string;
   public delete(): Promise<this>;
   public fetch(force?: boolean): Promise<this>;
-  public isThread(): this is AnyThreadChannel;
+  public isThread(): this is ThreadChannel;
```

Comment:
Wouldn't it be to better to keep this to the union type instead of the class to be able to narrow afterwards?

Reply:
Yes, I missed that. Thanks for pointing it out.